Eatery Blue/UI/EateryCard/EaterySmallCardView.swift

```diff
@@ -2,62 +2,102 @@
 //  EaterySmallCardView.swift
 //  Eatery Blue
 //
-//  Created by William Ma on 12/22/21.
+//  Created by Peter Bidoshi on 2/11/25.
+//  Originally created by William Ma on 12/22/21.
 //
 
+import Combine
 import EateryModel
 import UIKit
 
-class EaterySmallCardView: UIView {
+class EaterySmallCardView: UICollectionViewCell {
+
+    // MARK: - Properties (View)
```

Comment:
Nitpick: You used view as lowercase in other Marks but here its uppercase, I would try to keep things consistent